### DISCUSSION TITLE
cleanup of outdated issues

### DISCUSSION THREAD

**Discussion by n0xena** [demizer/archzfs#558] [General]
short and simple: I'm not a fan of a github repo being a place for abandoned issues - how about some cleanup?
most issues are either upstream problems, not vanilla arch or just outdated due to new versions of both zfs and kernel
so it would be nice if those who have permissions scan through the still open issues and check if they can be closed due to lack of activity or other reasons
thanks in advance

**Comment by stevleibelt** [comment]
If the main issue is missing manpower, I would volunteer to go through issues and close them when outdated.

I can verify that I am a regular person and that my intention to keep archzfs "up and running" in a good state if needed.
